tools/data-handler/src/containers/project.ts

```diff
@@ -233,6 +233,34 @@ export class Project extends CardContainer {
     return resources;
   }
 
+  /**
+   * Returns a new unique card key with project prefix (e.g. test_x649it4x).
+   * @returns a new card-key string
+   */
+  public async newCardKey(): Promise<string> {
+    // Create a key and check that there are no collisions with other keys in project
+    // Attempt max 10 times
+    for (let i = 0; i < 10; i++) {
```

Comment:
Make the ten a constant; name it `maxAttempts` (or similar), and remove the comment.

Reply:
Done in [852fc33](https://github.com/CyberismoCom/cyberismo/pull/127/commits/852fc3324963ad17dac30dd7baa1091d6b0f5c39)